app/models/motorcycle.rb

```diff
@@ -0,0 +1,8 @@
+class Motorcycle < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  has_many :reservations, foreign_key: 'motorcycle_id'
+
+  validates :name, presence: true, length: { minimum: 3, maximum: 50 }
+  validates :img_url, presence: true
+  validates :price, presence: true, numericality: { greater_than: 0 }
```

Comment:
There are a few more columns in the motorcycle table to be validated. Such as 'description', 'model_year', 'engine' and 'fuel_type'.

Reply:
@karayamanemre  Noted